src/Interpreters/SystemLog.cpp

```diff
@@ -152,19 +152,33 @@ std::shared_ptr<TSystemLog> createSystemLog(
     }
     else
     {
-        String partition_by = config.getString(config_prefix + ".partition_by", "toYYYYMM(event_date)");
+        /// ENGINE expr is necessary.
         engine = "ENGINE = MergeTree";
+
+        /// PARTITION expr is not necessary.
+        String partition_by = config.getString(config_prefix + ".partition_by", "toYYYYMM(event_date)");
         if (!partition_by.empty())
+        {
```

Comment:
Code style: This file is not using { } for single-line if/for/while statements. Could we please keep a consistent code style and omit { }?

Reply:
done.